bucket/typstyle.json

```
@@ -0,0 +1,28 @@
+{
+    "version": "0.13.17",
+    "description": "Beautiful and reliable typst code formatter.",
+    "homepage": "https://github.com/typstyle-rs/typstyle",
+    "license": "Apache-2.0",
+    "architecture": {

```

Suggest `vcredist` as this is a msvc compiled release

```suggestion
    "suggest": {
        "vcredist": "extras/vcredist2022"
    },
    "architecture": {
```